src/client/client-wrapper.ts

```diff
@@ -1,63 +1,27 @@
+import { ContactAwareMixin } from './mixins/contact-aware';
```

Comment:
- [ ] Recommend either **a)** updating `./mixins/index.ts` to include `contact-aware` mixin and then using that here (similar to how `UserAwareMixin` was set up below), or **b)** deleting `./mixins/index.ts` to prevent potential future confusion.